doc/devel/testing.rst

```diff
@@ -113,30 +124,20 @@ Writing a simple test
 Many elements of Matplotlib can be tested using standard tests. For
 example, here is a test from :mod:`matplotlib.tests.test_basic`::
 
-  from nose.tools import assert_equal
-
   def test_simple():
       """
       very simple example test
       """
-      assert_equal(1+1,2)
-
-Nose determines which functions are tests by searching for functions
-beginning with "test" in their name.
+      assert 1 + 1 == 2
 
-If the test has side effects that need to be cleaned up, such as
-creating figures using the pyplot interface, use the ``@cleanup``
-decorator::
+Pytest determines which functions are tests by searching for files whose names
+begin with ``"test_"`` and then within those files for functions beginning with
+``"test"`` or classes beginning with ``"Test"``.
 
-  from matplotlib.testing.decorators import cleanup
-
-  @cleanup
-  def test_create_figure():
-      """
-      very simple example test that creates a figure using pyplot.
-      """
-      fig = figure()
-      ...
+Tests that have side effects that need to be cleaned up, such as created
```

Comment:
There's no need for an example because the fixture gets used automatically; I've swapped the text around as suggested.